### DISCUSSION TITLE
Dowloading image in a post

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1610] [Ideas]
It is quite a pain to download images within a post.
There is the buton that looks like an "upload" button...BUT it will not download in media.... instead it will just put a relative link to your LOCAL folder....which obviousely no one wants ....since we are doing a website :)  
So this upload should actually upload to the gallery and put the ./file/media/.webp image in the link.
![image](https://github.com/user-attachments/assets/a94feac8-51cc-42af-abd6-b43098cfc0fd)

**Comment by hubyhuby** [comment]
WHat you have to do today
1) think to carefully avoid this "download button !
2) Go in  /tools and plugin / file manager
3) download the file there
4) Look at a previous working picture on your website
5) Copy the proper URL and append the file name
6) sync your website
7) check the image works properly. DONE !

**Comment by hubyhuby** [comment]
Here is the problem with upload of pictures (ubuntu 22.04) , with explaination in SOUND :

https://github.com/user-attachments/assets/58c2238d-5faf-4a38-b8da-c32a4e4e8e75

**Reply by bobmitro** [reply]
The Upload button (not confused with the Download button) is designed to upload images locally e.g. from your desktop. Once an image is uploaded, it, along with its responsive thumbnails (provided this option hasn't been disabled in the site settings), is placed in the media/post/ID directory associated with the specific post.

When the page is rendered, the path to the image remains local. However, when the site is synchronized, this path is updated to reflect the previously defined domain in the Server Settings.
